taskvine/src/worker/vine_cache.c

```diff
@@ -35,24 +38,19 @@ struct vine_cache {
 	char *cache_dir;
 };
 
-struct cache_file {
```

Comment:
The definition of `cache_file` should stay inside of `vine_cache` b/c we don't want other modules manipulating it outside of the API.